.pre-commit-config.yaml

```
@@ -16,6 +16,13 @@ repos:
         args:
           - "--profile"
           - "black"
+  - repo: https://github.com/doublify/pre-commit-rust
+    rev: master
+    hooks:
+    -   id: fmt
+        args: ['--manifest-path', './dask_planner/Cargo.toml', '--verbose', '--']
+    -   id: cargo-check
+        args: ['--manifest-path', './dask_planner/Cargo.toml', '--verbose', '--']
```

Think we should be able to get `clippy` running here with

```suggestion
        args: ['--manifest-path', './dask_planner/Cargo.toml', '--verbose', '--']
    -   id: cargo-clippy
        args: ['--manifest-path', './dask_planner/Cargo.toml', '--verbose', '--']
```